.github/workflows/main.yml

```
@@ -21,6 +21,22 @@ jobs:
           ansible-lint -r rules/ playbooks/*
           ansible-lint -r rules/ roles/*
 
+  molecule:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+      - name: Install molecule
+        run: pip install molecule
+      - name: Run tests
+        run: |
+          for folder in roles/*/molecule; do
+            pushd $(dirname $folder)
```

Generally not a fan of `folder`
```suggestion
          for directory in roles/*/molecule; do
            pushd $(dirname $directory)
```